src/styles/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+
+  height: 16px;
+  width: 16px;
+
+  background-image: url(../images/star-active.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5),
+.stars--5 .stars__star:nth-child(n+6) {
+  background-image: url(../images/star.svg);
+
+  background-repeat: no-repeat;
```

Comment:
Do you need this rule?

Reply:
It really is no need, thanks!
I think it was needed with the stars project and I copied this rule as well